scripts/amd/gemm/tune_gemm.py

```diff
@@ -674,7 +778,18 @@ def main():
         print(f"Unsupported dtype_a {args.dtype_a} or dtype_b {args.dtype_b} or dtype_c {args.dtype_c}")
         print("Supported types: ", list(name_to_tl_types.keys()))
         sys.exit(1)
-
+    rotating_buffer_size = args.rotating_tensor
+    bias_vector = args.bias_vector
+    icache_flush = args.icache_flush
+    if icache_flush:
+        if not is_hip_available():
+            print("************************************************************************************************")
+            print("  `icache-flush` is disabled for this run.")
+            print("  `icache-flush` needs python-hip module, which is unavailable.")
+            print("  python-hip module can be installed as:")
+            print("      `python3 -m pip install -i https://test.pypi.org/simple hip-python~=$rocm_version`")
```

Comment:
Can we detect rocm version automatically and put it here. Therefore, users can just copy paste the command to install hip-python

Reply:
Thanks, fixed